.neutrinorc.js

```diff
@@ -23,11 +23,19 @@ module.exports = {
          * Allow both '.jsx' and '.js' file extensions to contain JSX
          * ('.js' files are necessary for stories)
          */
-        'react/jsx-filename-extension': ['error', { 
-          'extensions': ['.js', '.jsx'] 
+        'react/jsx-filename-extension': ['error', {
+          'extensions': ['.js', '.jsx']
         }],
         /**
-         * 
+         * Allow exception to props spreading warning for divs.
+         * (so that the Tooltip component can apply to the div
+         *  wrapping children nodes, including custom components)
+         */
+        'react/jsx-props-no-spreading': ['error', {
+          'exceptions': ['div'],
+        }],
+        /**
+         * Allow 'dangerouslySetInnerHTML' for Markdown component.
          */
         'react/no-danger': ['off'],
```

Comment:
Rather than disabling this rule, we can do something like https://github.com/mozilla-frontend-infra/components/blob/46b814030ded93dc820ee7492e09f8ff1564ff13/src/components/Code/index.jsx#L40-L46.